src/components/Item/Item.js

```diff
@@ -0,0 +1 @@
+Item
```

Comment:
Codacy found an issue: ['Item' is not defined.](https://app.codacy.com/gh/WebHeroSchool/evafinal/pullRequest?prid=5874309)